cli/src/plugins/javadoc/index.ts

```diff
@@ -458,35 +525,43 @@ const makeClassDocPageBody: MakeBodyFunction = (args) => {
       makeBody: () =>
         makeTable(
           ["Modifier and Type", "Method and Description"],
-          doc.methods.map((doc) => [
-            [
-              md.text("\n"),
-              md.text(doc.modifiers),
-              md.text(" "),
-              project.linkToEntity(
-                doc.returnType.qualifiedTypeName,
-                doc.returnType.typeName
-              ),
-            ],
-            [
-              md.text("\n"),
-              md.paragraph([
-                project.linkToEntity(getCanonicalNameForMethod(doc), doc.name),
+          doc.methods
```

Comment:
Since sort() does modify the source, it might be more sane to copy the methods first:
```suggestion
          [...doc.methods]
```

Reply:
We actually rely on this for consistent ordering later on, in the method detail, as well. I think that this kind of side effect is undesirable enough that I should add a "processing" step earlier to sort all of the different collections of objects we plan do display to the user.

Reply:
note to self: for objects where ordering actually matters in advance, such as superclasses, just... don't sort it.

Reply:
Added a processing step earlier!